Makefile

```diff
@@ -1,22 +1,40 @@
 
 NAME = floop
 FILES = cmd/main.go cmd/cli.go
+BRANCH = $(shell git rev-parse --abbrev-ref HEAD)
+COMMIT = $(shell git rev-parse --short HEAD)
+VERSION = $(shell git describe | sed -e "s/^v//")
```

Comment:
i created the initial tag already so this should never be the case from this point forward.